packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -117,6 +124,20 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
 
   useManageDocumentTitle('Discussions');
 
+  useEffect(() => {
+    let timer: NodeJS.Timeout;
+
+    if (resizing) {
+      timer = setTimeout(() => {
+        setResizing(false);
+      }, 200); // Adjust delay as needed
+    }
+
+    return () => {
+      clearTimeout(timer);
+    };
+  }, [resizing]);
```

Comment:
so this supposed to be a fix? do we know why this helps and what is actually a root cause? This looks sooo hacky and tbh I do not understand it

Reply:
This is not the fix to the ticket. This was suppose to be here all along to control the `resizing` and `setResizing` 

Before the values were not resetting and they should've been. This was taken from `Sublayout.tsx` and how it's controlled on that level as well.